### PR TITLE
[CWS] Add dns events in activity dumps

### PR DESCRIPTION
**Pull request by Gui774ume**
### What does this PR do?

This PR adds the DNS event type in the activity dumps.

### Motivation

Expand the amount of dumped events.

### Reviewer's Checklist

- [x] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [x] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [x] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [x] Changed code has automated tests for its functionality.
- [x] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [x] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [x] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [x] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [x] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by djmitche — APPROVED** [review]
+1 pkg/config

**Review by kayayarai — COMMENTED** [review]
This PR doesn't appear to have customer facing docs. No Docs team review required.

File: pkg/security/probe/activity_dump.go [hunk 1]
```diff
@@ -911,6 +941,15 @@ func (pan *ProcessActivityNode) snapshotFiles(p *process.Process, ad *ActivityDu
 	return nil
 }
 
+// InsertDNSEvent inserts
+func (pan *ProcessActivityNode) InsertDNSEvent(evt *model.DNSEvent) bool {
+	if _, ok := pan.DNSNames[evt.Name]; ok {
+		return false
+	}
+	pan.DNSNames[evt.Name] = NewDNSNode(evt)
```
Comment (paulcacheux) [inline pkg/security/probe/activity_dump.go:949]: Aggregate by event type (A/AAA) and record type

Comment (Gui774ume) [reply pkg/security/probe/activity_dump.go:960]: 👍🏻

**Review by paulcacheux — APPROVED** [review]

**Review by spikat — APPROVED** [review]

**Pull request merged**